### PR TITLE
Vet clinic database: create animals table

### PR DESCRIPTION
**Pull request by UbdaNam**
## What does this PR do?

This PR is raised to implement the following features: 👍 :

- [x] Use GitFlow to introduce new features
- [x] Add vet-clinic schema
- [x] Add mock data to the database
- [x] Add some queries to retrieve data

## Description of the tasks completed 👇 

- [x] Add animals table
- [x] Setup PostgreSQL
- [x] The process involves installing PostgreSQL, creating a new database, defining tables using SQL commands, loading data into tables, configuring database settings, and connecting to the database. ☺️
  
## How should it be tested

Clone the repository
```
git clone https://github.com/UbdaNam/Vet-Clinic-db-query.git
```
Navigate to the folder
```
cd Vet-Clinic-db-query
```
Checkout the branch
```
git checkout Add-schema
```
Run app
open your terminal and run the following commands
```
  psql -U [enter your username here] -h localhost
  password:_"Enter your password"
  CREATE DATABASE vet-clinic;
```

Vet-clinic query result 1
Vet-clinic query result 2

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -244,48 +114,20 @@ Feel free to check the [issues page](../../issues/).
...
-
-  - [Answer_2]
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-<!-- LICENSE -->
-
 ## 📝 License <a name="license"></a>
 
 This project is [MIT](./LICENSE) licensed.
```
Comment (Qoosim) [inline README.md:131]: - [ ] The file you're referencing does not exist in your directory. Kindly create LICENSE file with the proper content in it and then reference it. That would prevent `404 - page not found` that's experiencing at this moment.

### Your MIT License page
license

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle:

Hi @UbdaNam, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
-  Good README file. :heavy_check_mark: 
-  Correct Gitflow. :heavy_check_mark: 




## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (UbdaNam) [reply README.md:131]: Thank you @Qoosim for your review 👍. I will make the change and ask for re-review

**Review by KanzaTahreem — APPROVED** [review]
# Approved 🥳

Hi @UbdaNam 🙋‍♀️

Great work implementing the suggested changes 💪 **_Your project is COMPLETE! There is nothing else to say other than... it's time to merge it! 🚀 Congratulations!_**



## Highlights 🎯
There are some excellent points that you have made perfectly. Keep up the great work! 👏

-  ✅ Table is created, perfect!
-  ✅ Great work using the right Gitflow 
-  ✅ Good job with queries

## Optional suggestions ❕
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

N/A

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by UbdaNam** [comment]
Thank you @KanzaTahreem for reviewing my PR

**Pull request merged**